### PR TITLE
fix: conditional on variable that could be None, and add logger debug to file

### PR DESCRIPTION
**Pull request by RouxAntoine**

### CODE REVIEW

File: db_migration.py [hunk 1]
```diff
@@ -15,6 +16,16 @@ def debug_user_list(users_list: List[User]):
 
 
 if __name__ == "__main__":
+    log_file_handler = logging.FileHandler(os.path.abspath("logs") + '/migration.log')
```
Comment (fxponscarme) [inline db_migration.py]: add timestamp in file name ?

File: migration/infrastructure/postgre_database.py [hunk 2]
```diff
@@ -1,4 +1,5 @@
 # Note: the module name is psycopg, not psycopg3
+from asyncio.log import logger
```
Comment (fxponscarme) [inline migration/infrastructure/postgre_database.py]: change import

**Review by fxponscarme — APPROVED** [review]

File: migration/infrastructure/postgre_database.py (hunk 2 above)
Comment (RouxAntoine) [reply migration/infrastructure/postgre_database.py]: resolve in commit 7b851f6

File: db_migration.py (hunk 1 above)
Comment (RouxAntoine) [reply db_migration.py]: fixed on commit 7b851f68

**Pull request merged**